src/client/actions/index.js

```diff
@@ -3,6 +3,9 @@ import {
   SEARCH_BY_SKILL,
   SEARCH_BY_NAME,
   SET_SKILLS,
+  ALL_LEARNERS,
+  CURRENT_LEARNERS,
```

Comment:
names of redux actions should have a verb associated to it. Just by looking at ALL_LEARNERS I don't have a sense of what this action does. Change it something like SET_FILTER_TO_ALL_LEARNERS